src/CoreWCF.Http/tests/ServiceContractNameTests.cs

```diff
@@ -16,6 +16,8 @@ namespace CoreWCF.Http.Tests
     public class ServiceContractNameTests
     {
         private readonly ITestOutputHelper _output;
+        private IWebHost _host;
```

Comment:
Why is this now a field and not a local variable?

Reply:
Having the host being a field has no impact on the tests of this file but does allow not having to update all the private methods in the file to pass the host as argument. I propose to open a follow up PR to refactor this test pattern globally as many "old" UT are written this way.